### PR TITLE
MNT Copy all generated files of the landing page to enable updates

### PR DESCRIPTION
**Pull request by eskayML**
#1176

copied all the files and set them under their respective folders

### CODE REVIEW

**Review by eskayML — COMMENTED** [review]
Seems like i made this changeunder the wrong branch

File: .circleci/config.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
       - run:
           name: "Copy landing_page.js"
           command: cp docs/static_landing_page/js/landing_page.js docs/_build/
-
+
```
Comment (romanlutz) [inline .circleci/config.yml:51]:
```suggestion
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -48,7 +48,6 @@ jobs:
       - run:
           name: "Copy landing_page.js"
           command: cp docs/static_landing_page/js/landing_page.js docs/_build/
```
Comment (romanlutz) [inline .circleci/config.yml:50]:
```suggestion
          command: cp docs/static_landing_page/js/landing_page.js docs/_build/

```

File: .circleci/push_doc.sh [hunk 3]
```diff
@@ -62,9 +62,38 @@ touch .nojekyll
 # landing page on fairlearn.org
 if [ "$CIRCLE_BRANCH" = "main" ]
 then
-    echo "Copying landing_page.js into js directory"
+    echo "Copying all the newly generated  files for the static landing page"
```
Comment (romanlutz) [inline .circleci/push_doc.sh:65]:
```suggestion
    echo "Copying all the newly generated files for the static landing page"
```

File: .circleci/push_doc.sh [hunk 4]
```diff
@@ -62,9 +62,38 @@ touch .nojekyll
...
     cp $GENERATED_DOC_DIR/../landing_page.js js/
+    #css
+    cp $GENERATED_DOC_DIR/../landing_page_style.css css/
+    #html
+    cp $GENERATED_DOC_DIR/../index.html  .
+    #fonts
+    cp $GENERATED_DOC_DIR/../SourceCodePro-Bold.ttf fonts/
+    cp $GENERATED_DOC_DIR/../SourceCodePro-Medium.ttf fonts/
+    cp $GENERATED_DOC_DIR/../SourceCodePro-Regular.ttf fonts/
+    #images
```
Comment (romanlutz) [inline .circleci/push_doc.sh:77]:
```suggestion
    # js
    cp $GENERATED_DOC_DIR/../landing_page.js js/
    # css
    cp $GENERATED_DOC_DIR/../landing_page_style.css css/
    # html
    cp $GENERATED_DOC_DIR/../index.html  .
    # fonts
    cp $GENERATED_DOC_DIR/../SourceCodePro-Bold.ttf fonts/
    cp $GENERATED_DOC_DIR/../SourceCodePro-Medium.ttf fonts/
    cp $GENERATED_DOC_DIR/../SourceCodePro-Regular.ttf fonts/
    # images
```

File: setup.cfg [hunk 5]
```diff
@@ -8,6 +8,9 @@ copyright-regexp = # Copyright \(c\) (Microsoft Corporation and )?Fairlearn cont
 # An update to pip has resulted in `pip install .` leaving a 'build' directory around
 # Also: 'pip freeze' does not report the pip version by default
 
+[tool:pytest]
+addopts = -Werror::DeprecationWarning -Werror::FutureWarning
```
Comment (romanlutz) [inline setup.cfg:12]: Should this be in a separate PR?

File: .circleci/push_doc.sh [hunk 6]
```diff
@@ -62,9 +62,38 @@ touch .nojekyll
...
+    cp $GENERATED_DOC_DIR/../Button_GH_Default.svg images/
+    cp $GENERATED_DOC_DIR/../Button_GH_Hover.svg images/
+    cp $GENERATED_DOC_DIR/../Discord_Color.svg images/
+    cp $GENERATED_DOC_DIR/../Discord_Inverse.svg images/
+    cp $GENERATED_DOC_DIR/../fairlearn_logo.svg images/
+    cp $GENERATED_DOC_DIR/../fairlearn-favicon.ico images/
+    cp $GENERATED_DOC_DIR/../fairlearn-meta-image.jpg images/
+    cp $GENERATED_DOC_DIR/../Github_Color.svg images/
+    cp $GENERATED_DOC_DIR/../Github_Inverse.svg images/
```
Comment (romanlutz) [inline .circleci/push_doc.sh:86]: can we replace all of these with 
```
cp $GENERATED_DOC_DIR/../*.svg images/
cp $GENERATED_DOC_DIR/../*.ttf images/

```
?

File: setup.cfg (hunk 5 above)
Comment (eskayML) [reply setup.cfg:12]: Ohh that's true , will remove that ASAP

File: .circleci/push_doc.sh (hunk 6 above)
Comment (eskayML) [reply .circleci/push_doc.sh:86]: Thanks , this looks way more efficient than mine😆

**Comment by eskayML** [comment]
Will the forced push I just performed cause any major problems ?

**Comment by romanlutz** [comment]
> Will the forced push I just performed cause any major problems ?

It's fine. If normale push doesn't work it's a sign that you should pull first since changes were made on the remote. I fixed some whitespace issues which are now back, but it's not a big deal.

I'm happy with this as is, but can you remove the blank lines and added whitespace?

File: .circleci/push_doc.sh [hunk 7]
```diff
@@ -62,9 +62,30 @@ touch .nojekyll
 # landing page on fairlearn.org
 if [ "$CIRCLE_BRANCH" = "main" ]
 then
-    echo "Copying landing_page.js into js directory"
+    echo "Copying all the newly generated files for the static landing page"
     ls $GENERATED_DOC_DIR/..
+
+    # js
+
```
Comment (romanlutz) [inline .circleci/push_doc.sh:69]: Nit: can we remove the black lines between description and command?

**Review by romanlutz — APPROVED** [review]
This looks fine to me, ready to merge except for whitespace fixes. Would be nice if another @fairlearn/fairlearn-maintainers takes a look, too.

File: .circleci/push_doc.sh (hunk 7 above)
Comment (eskayML) [reply .circleci/push_doc.sh]: Sure 💯

File: .circleci/config.yml [hunk 8]
```diff
@@ -50,7 +50,6 @@ jobs:
...
-
       - persist_to_workspace:
```
Comment (romanlutz) [inline .circleci/config.yml:53]:
```suggestion

      - persist_to_workspace:
```

**Review by adrinjalali — APPROVED** [review]
